queries/sql/highlights.scm

```diff
@@ -214,6 +255,7 @@
   "!="
   ">="
   ">"
+  "<>"
```

Comment:
Is it *just* used like <> to operate on something? Looks like brackets..but I'm probably wrong

Reply:
"not equal to" oeprator